flake.nix

```diff
@@ -80,16 +80,38 @@
       flake = false;
     };
 
-    #cardano-rosetta = {
-    #  url = "github:input-output-hk/cardano-rosetta";
-    #  flake = false;
-    #};
+    # cardano-rosetta = {
```

Comment:
why are we leaving this in?

Reply:
It was a placeholder that can go -- removing.